types/jquery.filtertable/index.d.ts

```diff
@@ -0,0 +1,144 @@
+/// <reference path="../jquery/jquery.d.ts" />
```

Comment:
The header must come first, anything else must come later.
Also, use `/// <reference types="jquery" />` instead.